cegs_portal/search/templates/search/v1/partials/_target_reg_effect.html

```diff
@@ -16,66 +18,87 @@
-    </tr>
-    {% empty %}
-    <tr>
-        <td>-</td>
-        <td>{{ regeffect.effect_size }}</td>
-        <td>{{ regeffect.direction }}</td>
-        <td>{{ regeffect.significance }}</td>
-        <td>
-            <a href="{% url 'search:experiment' regeffect.experiment.accession_id %}"
-            >{{ regeffect.experiment.name }}</a
-            >
-        </td>
-        <td><a href="{% url 'search:reg_effect' regeffect.accession_id %}">
-            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="svg-link-arrow bi bi-arrow-up-right-square-fill" viewBox="0 0 16 16">
-                <path fill-rule="evenodd" d="M15 2a1 1 0 0 0-1-1H2a1 1 0 0 0-1 1v12a1 1 0 0 0 1 1h12a1 1 0 0 0 1-1V2zM0 2a2 2 0 0 1 2-2h12a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V2zm5.854 8.803a.5.5 0 1 1-.708-.707L9.243 6H6.475a.5.5 0 1 1 0-1h3.975a.5.5 0 0 1 .5.5v3.975a.5.5 0 1 1-1 0V6.707l-4.096 4.096z"/>
-            </svg>
-            </a>
-        </td>
-    </tr>
-    {% endfor %}
-    {% endfor %}
-</table>
+                >
+            </td>
+            <td><a href="{% url 'search:reg_effect' regeffect.accession_id %}">
+                <svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="currentColor" class="svg-link-arrow bi bi-arrow-up-right-square-fill" viewBox="0 0 16 16">
+                    <path fill-rule="evenodd" d="M14 0a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V2a2 2 0 0 1 2-2zM5.904 10.803 10 6.707v2.768a.5.5 0 0 0 1 0V5.5a.5.5 0 0 0-.5-.5H6.525a.5.5 0 1 0 0 1h2.768l-4.096 4.096a.5.5 0 0 0 .707.707z"/>
+                </svg>
+                </a>
+            </td>
+        </tr>
+        {% endfor %}
+        {% endfor %}
+    </table>
+
+    <div class="pagination">
+        <span class="step-links">
+            {% if regeffects.has_previous %}
+                <a href="?page=1" hx-get="{% url 'search:target_effects' feature.accession_id %}?page=1" hx-target="#target-table">&laquo;>&laquo; first</a>
+                <a href="?page={{ regeffects.previous_page_number }}" hx-get="{% url 'search:target_effects' feature.accession_id %}?page={{ regeffects.previous_page_number }}" hx-target="#target-table"></a>>previous</a>
+            {% endif %}
+
+            <span class="current">
+                Page {{ regeffects.number }} of {{ regeffects.paginator.num_pages }}
+            </span>
+
+            {% if regeffects.has_next %}
+                <a href="?page={{ regeffects.next_page_number }}" hx-get="{% url 'search:target_effects' feature.accession_id %}?page={{ regeffects.next_page_number }}" hx-target="#target-table">next</a>
+                <a href="?page={{ regeffects.paginator.num_pages }}" hx-get="{% url 'search:target_effects' feature.accession_id %}?page={{ regeffects.paginator.num_pages }}" hx-target="#target-table">last &raquo;</a>
+            {% endif %}
+        </span>
+    </div>
+    </div>
+
+</div>
+
+{% include 'search/v1/partials/_row_click_js.html' %}
```

Comment:
Doesn't this not work anymore?

Reply:
Will remove but it might be nice to keep the row click but adjust it in a way for the user to right click to view on another tab etc. The team liked it and suggested it--hopefully in the future we can add it back.

Reply:
Well, delete the reference for now please.

Reply:
Should be deleted in the most recent commit

Reply:
it's still in _reg_effect.html